docs/articles/infrastructure/ci-cd/jenkins.rst

```diff
@@ -60,7 +60,7 @@ Then you can open this jenkins instance in your browser and put this password th
 
    http://localhost:8080/jenkins/
 
-After that you can also set up a Nginx instance as a proxy so you don't need the port number in the URL, but you don't need to do that on your development machine.
+After that you can also set up a NGINX instance as a proxy so you don't need the port number in the URL, but you don't need to do that on your development machine.
```

Comment:
I think the preferred spelling here is `nginx`, at least that's how it's written on their own home page.

Reply:
You're probably right.. I was looking at this: https://www.nginx.com/resources/wiki/ <-- But https://nginx.org/en/ uses all lower case.